Chrysalis-Papilio/Soen-490-Papilio
# User profile page fix

**User Profile Page**

What to test:
- Verify that the User Profile Page can be accessed from the nav bar like Account -> My Profile
- Verify that the User Profile Page appears correctly on your device/emulator
- Verify that the Horizontal Scroll Bar works

How to do it:
- Simply run the app, log in using whatever method you wish
- Then tap "Account" on the Nav Bar
- Then tap the "My Profile" button

Testing:
- run the tests locally using the command ./gradlew connectedAndroidTest
